public_dropin_environments/python3_sklearn_fedramp_compliant/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# This is a private chain-guard development image that is stored in DataRobot's private registry.
+# Replace it with your own development chain-gaurd image if you build your own.
+ARG BASE_ROOT_IMAGE=datarobotdev/mirror_chainguard_datarobot.com_python-fips:3.11-dev
+FROM ${BASE_ROOT_IMAGE} AS build
+
+USER root
+
+# Fix these CVEs: CVE-2025-0395, CVE-2024-13176, CVE-2025-0395
+RUN apk update && apk upgrade --available && \
```

Comment:
This looks a little weird to me. This is the build image, why do we care what libs are installed into it? Are we statically linking against these when we build pip wheels (which then get copied to prod image)?

Reply:
It was done to be able to install certain libraries to fix existing CVEs (see the comment above it).
But maybe, it should not be our concern?

Reply:
Yeah I think artifacts inherited from the base image should be the responsibility of the base image to update (our internal tooling is getting better at detecting the base image and not reporting duplicate CVEs for all downstream images). While your code here will make the image CVE free, I think it adds extra maintenance for us (i.e. to remove this code when the base has updated their deps or update them when there is a future CVE found in one of these libraries).